### ISSUE TITLE
Baseline code release ?

### ISSUE THREAD

**Issue by trungpham2606** [iShape/iShape.github.io#1]
Dear @DIYer22 
I just had read your paper. It's such a great work.
When will you release the baseline code for training and testing ?

**Comment by DIYer22** [comment]
We are advancing open source work, the "building iShape" code is already released. 
But the baseline code is still in progress. It should be released within two weeks at the latest.

Thanks for attention!

**Comment by trungpham2606** [comment]
Oh, so glad to hear it from you. 
Iam looking forward to testing it ^^

**Comment by trungpham2606** [comment]
Dear @DIYer22 
How is the release progress ^^

**Comment by DIYer22** [comment]
@trungpham2606  The progress was interrupted by preparing rebuttals for NIPS and other things. We plan to advance open-source baseline code after 07/24. Sorry for the delay. We will notify you as soon as we release the baseline code.

**Comment by trungpham2606** [comment]
Thank you for your prompt response @DIYer22

**Comment by trungpham2606** [comment]
Hello, is everything oke @DIYer22 ?

**Comment by DIYer22** [comment]
We try to allocate more time to push the progress, Really really sorry for the delay again!

**Comment by trungpham2606** [comment]
@DIYer22 
I have a question that beside irregular objects , with normal shape objects, will the network still work well ?
And what is the inference time for 1 image ?

**Comment by qianyizhang** [comment]
just a quick question, how is the irregular shape kernel implemented?
did you write new cuda ops or using F.gird_sample?

**Comment by DIYer22** [comment]
@qianyizhang  No, just by numpy. kernel is used on GT generation and build graph. No need to do it on the GPU

**Comment by qianyizhang** [comment]
> @qianyizhang No, just by numpy. kernel is used on GT generation and build graph. No need to do it on the GPU

1. when calculcating GT, since its not regular grid, how did you effectively aggregate the result? or you simply does a big for loop then stack?
2. in training/inference, do you simply make a vector prediction which is supervised by the GT affinity? I thought you make aggregating of feature (by kernel) then calculate their pair-wise similarity/distance, which is then supervised by GT

**Comment by robbyneven** [comment]
Any ETA on baseline code release yet?

**Comment by invincible-28** [comment]
When will the baseline code be released? Please let us know.

**Comment by SEUZTh** [comment]
2022.
Any ETA on baseline code release yet?

**Comment by mchaniotakis** [comment]
Thank you for creating this segmentation method, could you please update us on the status of the code release? Thank you.

**Comment by xiongsheng001** [comment]
It's been a long time since you promised to open the source code.

**Comment by Asthestarsfalll** [comment]
Have you accidentally deleted your code?

**Comment by tjw1020** [comment]
I just had read your paper. It's such a great work.
When will you release the baseline code for training and testing ?
